action/evm.go

```diff
@@ -78,8 +78,6 @@ func EthereumConfig(chainID string) *ethparams.ChainConfig {
 		MuirGlacierBlock:    big.NewInt(0),
 		BerlinBlock:         big.NewInt(0),
 		LondonBlock:         big.NewInt(0),
-
-		EWASMBlock: nil,
```

Comment:
why?

Reply:
the new go-ethereum doenst support this anymore, so I removed it